pkg/machines/components/networks/createNetworkDialog.jsx

```diff
@@ -0,0 +1,608 @@
+            <MachinesConnectionSelector id='network-dialog-connection'
+                dialogValues={dialogValues}
+                onValueChanged={onValueChanged}
+                loggedUser={loggedUser} />
+        </React.Fragment>
+    );
+};
+
+const NetworkNameRow = ({ onValueChanged, dialogValues, validationFailed }) => {
+    const validationState = validationFailed.name ? 'error' : undefined;
+
+    return (
+        <React.Fragment>
+            <label className='control-label'>
+                {_("Name")}
+            </label>
+            <FormGroup validationState={validationState} controlId='name'>
+                <input
+                   id='network-create-name'
+                   type='text'
+                   placeholder={_('Unique Network Name')}
+                   value={dialogValues.name}
+                   onChange={e => onValueChanged('name', e.target.value)}
+                   className='form-control' />
+                { validationState == 'error' &&
+                <HelpBlock>
+                    <p className='text-danger'>{validationFailed.name}</p>
+                </HelpBlock> }
+            </FormGroup>
+        </React.Fragment>
+    );
+};
+
+const NetworkForwardModeRow = ({ onValueChanged, dialogValues }) => {
+    const forwardModes = ['nat', 'open', 'none'];
+
+    return (
+        <React.Fragment>
+            <label className='control-label'>
+                {_("Forward Mode")}
+            </label>
+            <Select.Select id='create-network-forward-mode'
+                           initial={dialogValues.forwardMode}
+                           onChange={value => onValueChanged('forwardMode', value)}>
+                { forwardModes.map(mode => {
+                    return (
+                        <Select.SelectEntry data={mode} key={mode}>
+                            {rephraseUI('networkForward', mode)}
+                        </Select.SelectEntry>
+                    );
+                })
+                }
+            </Select.Select>
+        </React.Fragment>
+    );
+};
+
+const NetworkDeviceRow = ({ onValueChanged, dialogValues, vms, nodeDevices, networks, validationFailed }) => {
+    const validationState = validationFailed.physicalDevice ? 'error' : undefined;
+    const devices = getNetworkPhysicalDevices(vms, nodeDevices, networks);
+
+    // TODO Implement "Any physical device" option later
+    return (
+        <React.Fragment>
+            <label className='control-label'>
+                {_("Device")}
+            </label>
+            <FormGroup validationState={validationState} controlId='physicalDevice'>
+                <Select.Select id='create-network-physical-device'
+                               enabled={dialogValues.physicalDevice !== undefined}
+                               initial={dialogValues.physicalDevice !== undefined || "No Physical Device Available"}
+                               onChange={value => onValueChanged('physicalDevice', value)}>
```

Comment:
This is wrong. I guess you wanted to use short if then else. Aka condition ? value : other-value.

Reply:
Also this was breaking the select and you where not able to change from one device to another ;)